src/overlays/actors/ovl_En_Gs/z_en_gs.c

```diff
@@ -438,7 +438,7 @@ void func_80A4F13C(EnGs* this, PlayState* play) {
         }
     }
     if (this->unk_19F == 4) {
-        tmpf1 = Math_SmoothStepToF(&this->unk_1E8, this->unk_1EC, 0.8f, 16384.0f, 3640.0f);
+        tmpf1 = Math_SmoothStepToF(&this->unk_1E8, this->unk_1EC, 0.8f, DEG_TO_BINANG2(90.0f), 3640.0f);
```

Comment:
this one the "left hand side" is the argument, typed as f32, surely it can be `DEG_TO_BINANG` ?

Reply:
Yeah this one is fine, I used DEG_TO_BINANG2 here for consistency with DEG_TO_BINANG2(20.0f) below (which is required for matching)

Reply:
we could also just leave this as 16384.0f